src/pages/patientView/SampleManager.tsx

```diff
@@ -58,7 +58,8 @@ export function sortSamples(samples: Array<ClinicalDataBySampleId>,
         let eventOrdering = collectionDayMap[id];
 
         // 2. if cases have derived normalized case types, put primary first
-        let sampleTypeIndex = sampleTypeOrdering.indexOf(clinicalDataLegacyCleanAndDerived[id].DERIVED_NORMALIZED_CASE_TYPE);
+        const caseType = clinicalDataLegacyCleanAndDerived[id].DERIVED_NORMALIZED_CASE_TYPE;
+        let sampleTypeIndex = caseType ? sampleTypeOrdering.indexOf(caseType.toLowerCase()) : -1;
```

Comment:
instead of first assigning -1 and then correcting it on ln 64, could we just assing sampleTypeOrdering.lenth in the ternary?

Reply:
good call

Reply:
oh I just realized, that doesn't capture the case when `sampleTypeOrdering.indexOf(caseType.toLowerCase())` is -1